californium-core/src/main/java/org/eclipse/californium/core/coap/OptionSet.java

```diff
@@ -135,6 +135,7 @@ public void clear() {
 		block2 = null;
 		observe = null;
 		oscore = null;
+
```

Comment:
I still prefer, if this file would not be changed only to format code, 
Please replace it with the previous one.

Reply:
Sorry.